doc.go

```diff
@@ -382,6 +388,23 @@ of pigeon and should not be used nor modified. Those keys are treated as
 internal implementation details and therefore there are no guarantees given in
 regards of API stability.
 
+Left recursion
+
+With options -support-left-recursion pigeon supports left recursion. E.g.:
+	expr = expr '*' term
+Supports indirect recursion:
+	A = B / D
+	B = A / C
+Supports priorities:
+	expr = expr '*' term / expr '+' term
```

Comment:
Fixed

This wording is really better. And it is important because the papers and implementation to which I gave links do not support ordered choice in the rules with left recursion.

Reply:
Sorry, really everything works as it is by default. I will remove this piece.

Reply:
Fixed